GLOCKFILE

```diff
@@ -23,7 +23,7 @@ github.com/chzyer/readline dc5da28fbf5287157fcd4719c794b59cd8852115
 github.com/client9/misspell 4f5982f81a18693b85c9926407fc4f61102ee0e7
 github.com/cockroachdb/c-jemalloc cc6099c89203d25620682bb45844bdade48559b9
 github.com/cockroachdb/c-protobuf a6f3c8c4bc8a36a44d7c876c37a3198ba3923316
-github.com/cockroachdb/c-rocksdb 7d9da2c20d8f436e36dc0114b1a516f386e47009
+github.com/cockroachdb/c-rocksdb 6774b5fa42266d97d3a41770990c06370a24f3c6
```

Comment:
No, that one is out of date. You can see the branch: https://github.com/cockroachdb/c-rocksdb/tree/pmattis/prefix-same-as-start